config.py

```diff
@@ -0,0 +1,12 @@
+
+# Configuation file
+import os
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+# Attributes of the database:
+CSRF_ENABLED = True
```

Comment:
A couple of comments on this file:
1. CSRF enabled isn't a database attribute
2. Secret keys should not live in github. I'd google "how to store database secret keys" for a better solution
3. This file should probably be loading the config.cfg file, and use that to set these attributes. Otherwise your going to end up in frustrating situation where you change your config file but don't see the expected changes in your application.